### PR TITLE
fix 404 for post-consumption-example

### PR DESCRIPTION
**Pull request by ghost**
## Proposed change
Fixed 404 of `post-consumption-example.sh`. Still pointed to Jonas' repository.

Fixes # (issue)

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Other (please explain)

## Checklist:

- [x] I have read & agree with the [contributing guidelines](https://github.com/paperless-ngx/paperless-ngx/blob/main/CONTRIBUTING.md).
- [ ] If applicable, I have tested my code for new features & regressions on both mobile & desktop devices, using the latest version of major browsers.
- [ ] If applicable, I have checked that all tests pass, see [documentation](https://paperless-ngx.readthedocs.io/en/latest/extending.html#back-end-development).
- [ ] I have run all `pre-commit` hooks, see [documentation](https://paperless-ngx.readthedocs.io/en/latest/extending.html#code-formatting-with-pre-commit-hooks).
- [ ] I have made corresponding changes to the documentation as needed.
- [ ] I have checked my modifications for any breaking changes.

### CODE REVIEW

File: docs/advanced_usage.rst [hunk 1]
```diff
@@ -199,7 +199,7 @@ Troubleshooting:
 - Check your script's permission e.g. in case of permission error ``sudo chmod 755 post-consumption-example.sh``
 - Pipe your scripts's output to a log file e.g. ``echo "${DOCUMENT_ID}" | tee --append /usr/src/paperless/scripts/post-consumption-example.log``
 
-.. _post-consumption-example.sh: https://github.com/jonaswinkler/paperless-ngx/blob/master/scripts/post-consumption-example.sh
+.. _post-consumption-example.sh: https://github.com/paperless-ngx/paperless-ngx/blob/master/scripts/post-consumption-example.sh
```
Comment (tribut) [inline docs/advanced_usage.rst:202]:
```suggestion
.. _post-consumption-example.sh: https://github.com/paperless-ngx/paperless-ngx/blob/main/scripts/post-consumption-example.sh
```

**Review by tribut — CHANGES_REQUESTED** [review]
Thanks!

Branch has been renamed to main, otherwise this obviously makes sense.

File: docs/advanced_usage.rst (hunk 1 above)
Comment (fantasticle) [reply docs/advanced_usage.rst:202]: makes sense - auto-forwarding from `master` to `main` made me miss that point, sorry about that.

**Review by tribut — APPROVED** [review]
LGTM 🙂

**Review by stumpylog — APPROVED** [review]

**Pull request merged**